### PR TITLE
Restructure services/available-env-vars.md

### PR DESCRIPTION
**Pull request by Tonnu**
Renamed to `linking-services.md`, restructured  and rewrote some paragraphs for clarity.

### CODE REVIEW

File: content/docs/services/linking-services.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+### Using the environment variables
+To use these env vars in your application you
+would simply query the environment for the variables you are looking for. In
+this example, you would want the IP and the port elasticsearch is running on. In
+Go this would look something like this:
+
+```golang
+fmt.Println(“IP:", os.Getenv("ELASTICSEARCH_PORT_9300_TCP_ADDR”))
+fmt.Println(“PORT:", os.Getenv("ELASTICSEARCH_PORT_9300_TCP”))
```
Comment (hatchan) [inline content/docs/services/linking-services.md]: The env var should be `ELASTICSEARCH_PORT_9300_TCP_PORT`

File: content/docs/services/linking-services.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+Dockerlinks work by exposing the **service container’s** environment variables to
+the **current environment**. This happens for every service listed in the
+`services:` section. For each service, Docker will expose **four** distinct
+environment variables:
+
+```no-highlight
+<name>_PORT_<port>_<protocol>_ADDR
+<name>_PORT_<port>_<protocol>_PORT
+<name>_PORT_<port>_<protocol>_PROTO
+<name>_PORT
```
Comment (hatchan) [inline content/docs/services/linking-services.md]: Probably best to leave this one out. As it just a bit confusing what it contains.

Had to look it up, so if anyone else is curious: 

> This variable contains the URL of the source container’s first exposed port. The ‘first’ port is defined as the exposed port with the lowest number.

File: content/docs/services/linking-services.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Wercker uses [dockerlinks](https://docs.docker.com/userguide/dockerlinks
+/#environment-variables) to enable interaction between containers.
+Every service that is specified in your **wercker.yml** will be linked to the
+main box, where your application lives.
+
+### Exposed environment variables
+Dockerlinks work by exposing the **service container’s** environment variables to
+the **current environment**. This happens for every service listed in the
+`services:` section. For each service, Docker will expose **four** distinct
```
Comment (hatchan) [inline content/docs/services/linking-services.md]: Rewrite to account for multiple ports to:

```
For each exposed port in each service, Docker will expose **three** distinct environment variables: 
```

File: content/docs/services/linking-services.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+environment variables:
+
+```no-highlight
+<name>_PORT_<port>_<protocol>_ADDR
```
Comment (hatchan) [inline content/docs/services/linking-services.md]: Also might be worth mentioning how this `name` is generated.

**Comment by hatchan** [comment]
Tiny things, :+1:

**Comment by Tonnu** [comment]
This has been open for quite a while, merging it in.

**Pull request merged**